docs/guide_about_files_generated.md

```diff
@@ -0,0 +1,305 @@
+Guide
```

Comment:
@daren-thomas: I updated the *.rst file. Please take a look at it. I will delete the `markdown` file once we finalize the `rst` file. Thanks